tools/calyx-writer/lib.rs

```diff
@@ -373,10 +376,13 @@ impl CalyxWriter for Guard {
             Guard::True => {}
             Guard::Port(port) => port.write(f)?,
             Guard::And(lhs, rhs) => {
+                write!(f, "(")?;
                 lhs.write(f)?;
                 write!(f, " & ")?;
                 rhs.write(f)?;
+                write!(f, "(")?;
```

Comment:
I think this paren is backwards? or I'm just confused

Reply:
Lol